wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
Coudn't you then use reshape() on string's .data? To get its copy as a UInt8 buffer of required size?
I'm sorry that I don't know nothing about Julia, just guessing.